lib/public_key/src/pubkey_cert.erl

```diff
@@ -2195,3 +2196,13 @@ otp_cert(#'OTPCertificate'{} = Cert) ->
     Cert;
 otp_cert(#cert{otp = OtpCert}) ->
     OtpCert.
+
+key_params(#'OTPTBSCertificate'{signature =
```

Comment:
indeed the formatting is now worse.
why not do it something like?

```
key_params(#'OTPTBSCertificate'{
              signature = #'SignatureAlgorithm'{
                             algorithm = ?'id-RSASSA-PSS',
                             parameters = KeyParams}},
           KeyParams0) when KeyParams0 == asn1_NOVALUE; KeyParams0 == 'NULL' ->
    %% Sometimes parameters may be missing in issuer's "SubjectPublicKeyInfo"
    %% but included in the certs "SignatureAlgorithm" for RSA PSS signatures.
    KeyParams;
key_params(_, KeyParams) ->
    KeyParams.
```

Reply:
Well I settled for

```
key_params(#'OTPTBSCertificate'{signature =
                                    #'SignatureAlgorithm'{algorithm =
                                                              ?'id-RSASSA-PSS',
                                                          parameters = KeyParams}},
           KeyParams0) when KeyParams0 == asn1_NOVALUE;
                                          KeyParams0 == 'NULL' ->
    %% Sometimes parameters may be missing in issuer's
    %% "SubjectPublicKeyInfo" but included in the certs
    %% "SignatureAlgorithm" for RSA PSS signatures.
    KeyParams;
key_params(_, KeyParams) ->
    KeyParams.

```

I guess I feel it is unnatural to break already after "{"  in the record name, but that looks ok too.